src/Nri/Ui/Page/V3.elm

```diff
@@ -0,0 +1,223 @@
+module Nri.Ui.Page.V3 exposing
+    ( DefaultPage, broken, blocked, notFound, noPermission
+    , RecoveryText(..)
+    )
+
+{-| A styled NRI issue page!
+
+@docs DefaultPage, broken, blocked, notFound, noPermission, RecoveryText(..)
+
+-}
+
+import Css exposing (..)
+import Html.Styled as Html exposing (Html)
+import Html.Styled.Attributes as Attributes
+import Nri.Ui.Button.V5 as Button
+import Nri.Ui.Text.V2 as Text
+
+
+{-| The default page information is for the button
+which will direct the user back to the main page of
+the SPA. Specify it's name and the message which will
+navigate to the page.
+-}
+type alias DefaultPage msg =
+    { link : msg
+    , recoveryText : RecoveryText String
+    }
+
+
+{-| ReturnTo just needs the name of the page the user
+will be returned to. Reload displays default text to reload
+the current page. Custom is to add flexibility to the button.
+-}
+type RecoveryText a
+    = ReturnTo a
+    | Reload
+    | Custom a
```

Comment:
☁️ Remove the type variable? It's always going to be `String`. If we change the definition of `recoveryText` above it would be a breaking change, and we'd need to create a `V4` module anyway.

Reply:
My other thought about this was to smush `DefaultPage` and `RecoveryText` into a single custom type:

```elm
type Config msg
    = ReturnTo String msg
    | Reload String msg
    | Custom String msg

-- ... and, later in the file, rename the
-- non-public Config to ViewConfig, say.
```

Just a thought, not a request.